aces/Loris
# [candidate_list] Add Derived Age to candidate list

## Brief summary of changes

Replace EDC exposure in candidate_list with derived age values and add the Derived Age as a new column:

- Display `Derived Age`.
- Display `EDC Age` instead of raw EDC when `useEDC` is enabled.
- Derive ages in years/months.
- Update candidate_list filters, integration tests, README/test plan, and translations.
- Keep the `EDC` acronym untranslated while translating only “Age”.

#### Testing instructions (if applicable)

1. Run `make`.
2. If front-end assets are not rebuilt by `make`, run `target=candidate_list npm run compile`.
3. Navigate to `/candidate_list` and verify:
   - The table shows `Derived Age`.
   - The table shows `EDC Age` instead of EDC when `useEDC` is enabled.
   - The table shows `DoB` when `useDoB` is enabled.
   - The advanced filters include `Derived Age` and `EDC Age`, as well as DoB
4. Confirm translations show `EDC` unchanged, for example `Âge EDC`, `EDC 年齢`, `EDC 年龄`, `EDC आयु`.

- Related to: https://github.com/aces/Loris/issues/8357
